plugins/org.python.pydev.ast/src/org/python/pydev/ast/codecompletion/PyStringCodeCompletion.java

```diff
@@ -149,17 +156,42 @@
     /**
      * Needed interface for adding the completions on a request
      * @throws MisconfigurationException
+     * @throws PythonNatureWithoutProjectException
+     * @throws IOException
      */
     @Override
     public TokensOrProposalsList getCodeCompletionProposals(CompletionRequest request) throws CoreException,
-            BadLocationException, MisconfigurationException {
+            BadLocationException, MisconfigurationException, IOException, PythonNatureWithoutProjectException {
+
         List<ICompletionProposalHandle> completionProposals = new ArrayList<>();
         request.showTemplates = false; //don't show templates in strings
         fillWithEpydocFields(request, completionProposals);
 
         TokensOrProposalsList ret = new TokensOrProposalsList();
         if (completionProposals.size() == 0) {
             //if the size is not 0, it means that this is a place for the '@' stuff, and not for the 'default' context for a string.
+            IDocument doc = request.doc;
+            ITypedRegion partition = ((FastPartitioner) PyPartitionScanner.checkPartitionScanner(doc))
+                    .getPartition(request.documentOffset);
+            String partitionType = partition.getType();
+
+            if (partitionType.equals(IPythonPartitions.PY_MULTILINE_FSTRING1)
```

Comment:
Please create a `Set<String>` on `IPythonPartitions.F_STRING_PARTITIONS` and then you can check if `IPythonPartitions.F_STRING_PARTITIONS.contains(partitionType)`.